### ISSUE TITLE
bug: gtk-enable = true breaks home-manager-"$USER" service

### ISSUE THREAD

**Issue by NotAShelf** [nix-community/home-manager#3429]
### Are you following the right branch?

- [X] My Nixpkgs and Home Manager versions are in sync

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Issue description

For the past 3 days or so, home-manager has been failing to start the service with the following error log in `journalctl -xeu home-manager-<username>`. 

https://p.frzn.dev/tik/

I have been dissecting my config over and over again, but could not find a possible cause for the isssue nor any plausible reproduction steps.  I have also tried rolling back to past builds of my system, which unfortunately did not help; which leads me to believe it may be introduced in a recent home-manager commit. With that said, I have a rising suspicion that it is the [zsh module](https://github.com/NotAShelf/dotfiles/blob/nixos/modules/home/terminal/shell/default.nix) as zsh seems to be the only noticeable config that hasn't been copied to ~/.config but commeting that module out does not help with the issue.

My nixos configuration can be found [here](https://github.com/notashelf/dotfiles).

### System information

```markdown
- system: `"x86_64-linux"`
 - host os: `Linux 6.0.7, NixOS, 22.11 (Raccoon), 22.11.20221109.872fcee`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.11.0`
 - nixpkgs: `/etc/nix/flake-channels/nixpkgs`
```

**Comment by NotAShelf** [comment]
After some more extensive testing, and rolling back to versions where home-manager used to work the issue still persists. Which leads me to believe it is, in fact, home-manager that is the source of the problem.

**Comment by NotAShelf** [comment]
Some more debugging later, I have pinpointed the issue to the GTK module. Namely, having `gtk.enable = true;` in your home-manager config causes home-manager-"$USER" service to fail on startup

**Comment by NotAShelf** [comment]
possibly relevant to #3405

**Comment by NotAShelf** [comment]
closed by #3405
